packages/enricher/src/helpers.ts

```diff
@@ -0,0 +1,86 @@
+import {
+  ontologyVersionIdentifier,
+  ontologyUrl,
+  AdditionalType,
+  Enrichment,
+} from './definitions';
+import {defu} from 'defu';
+import type {Resource} from 'rdf-object';
+
+export function fromAboutTypeToClass(additionalType: AdditionalType) {
+  let className;
+  if (additionalType === AdditionalType.Description) {
+    className = 'Description';
+  } else if (additionalType === AdditionalType.Name) {
+    className = 'Name';
+  } else {
+    throw new TypeError(`Unknown type: "${additionalType}"`);
```

Comment:
Only `Name` and `Description` are supported here, but there are more additional types, are there rest unsupported?

Reply:
I've just updated the code - the other types are now also being supported.